sections/extras.tex

```diff
@@ -10,6 +10,21 @@ \section{Extras}
   \end{itemize}
 \end{frame}
 
+\subsection{What to commit and what not to commit}
+\begin{frame}[fragile]
+  \slidetitle
+  It's best practice to not commit:
+  \begin{itemize}
+    \item Generated Binaries
+    \item Large files (Use \lstinline{git-lfs})
+    \item Your personal editor configuration
+    \item ...
```

Comment:
Passwords and Secrets :)

Reply:
but I store all my ssh keys in a git repo :stuck_out_tongue: